app/controllers/api/subscriptions_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::SubscriptionsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    if params[:stripeToken]
+      begin
+        customer = Stripe::Customer.list(email: params[:email]).data.first
+        customer = Stripe::Customer.create({ email: params[:email], source: params[:stripeToken] }) unless customer
```

Comment:
`current_user.email` will be more appropriate to use here.

Reply:
Updated this

Reply:
Looks good!

Reply:
Verified